changelogs/fragments/nxos_facts_fix_backport_fix.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- nxos_facts fix <https://github.com/ansible/ansible/pull/57009>
```

Comment:
Note that changelog entries are best when they tell the user what symptoms they may be seeing which have been fixed.  (Not a blocker; just something to think about when you write your next one).